molecule/rbac-mds-kerberos-debian/molecule.yml

```diff
@@ -261,6 +263,17 @@ provisioner:
         custom_java_path: /opt/jdk17 # Use custom Java 17
 
       mds:
+
+        # necessary configs in MDS server for sso in C3
+        sso_groups_claim: groups
+        sso_sub_claim: sub
+        sso_issuer_url: https://login.microsoftonline.com/0893715b-959b-4906-a185-2789e1ead045/v2.0
+        sso_jwks_uri: https://login.microsoftonline.com/0893715b-959b-4906-a185-2789e1ead045/discovery/v2.0/keys
+        sso_authorize_uri: https://login.microsoftonline.com/0893715b-959b-4906-a185-2789e1ead045/oauth2/v2.0/authorize
+        sso_token_uri: https://login.microsoftonline.com/0893715b-959b-4906-a185-2789e1ead045/oauth2/v2.0/token
+        sso_client_id: e674334a-de0d-4b28-a9ed-a628285ce7c4
+        sso_client_password: pqb8Q~uNc4AVwLZgXMCE_ONJ6jIEbqnWzp4U_c5c
```

Comment:
where do we get these client id and password from ?? 
is it ok to keep it hardcoded in the code ?